conf/machine/imx8mq-evk.conf

```diff
@@ -4,40 +4,47 @@
 #@DESCRIPTION: Machine configuration for NXP i.MX 8M Quad Evaluation Kit
 #@MAINTAINER: Jun Zhu <junzhu@nxp.com>
 
-MACHINEOVERRIDES =. "mx8:mx8m:mx8mq:"
+MACHINEOVERRIDES =. "imx-boot-container:mx8:mx8m:mx8mq:"
 
 require conf/machine/include/imx-base.inc
 require conf/machine/include/arm/armv8a/tune-cortexa53.inc
 
-IMX_DEFAULT_BSP = "nxp"
+IMX_DEFAULT_BSP = "mainline"
 
-MACHINE_FEATURES += "pci wifi bluetooth optee bcm43455 bcm4356"
-MACHINE_FEATURES:append:use-nxp-bsp = " bcm4359"
+# Mainline BSP defaults to "generic" cortexa53 configuration,
+# adjust it here to include crypto extension which enables
+# inline NEON and FPU code generation
+DEFAULTTUNE:use-mainline-bsp = "cortexa53-crypto"
+
+MACHINE_FEATURES += "pci wifi bluetooth bcm43455 bcm4356"
+MACHINE_FEATURES:append:use-nxp-bsp = " optee bcm4359"
 
 MACHINE_SOCARCH_FILTER:append:mx8mq = " virtual/libopenvg virtual/libgles1 virtual/libgles2 virtual/egl virtual/mesa virtual/libgl virtual/libg2d"
 
+KERNEL_DEVICETREE_BASENAME = "imx8mq-evk"
```

Comment:
What is the goal of this variable? it doesn't vary across nxp and mainline so it adds a indirection.

Reply:
Correct, will drop it.